supporting_scripts/analysis-of-endpoint-connections/src/main/java/de/tum/cit/endpointanalysis/EndpointInformation.java

```diff
@@ -0,0 +1,126 @@
+package de.tum.cit.endpointanalysis;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class EndpointInformation {
+
+    @JsonProperty
```

Comment:
Can you check if we really need to add the `@JsonProperty` annotation? In my experience, this is only used if we want to define a custom name for the json property